### PR TITLE
Graph

### PR DESCRIPTION
**Pull request by LeoraBennett**

### CODE REVIEW

**Comment by LeoraBennett** [comment]
my values are wrong in the table, what can i do to fix it?

**Comment by andrewoid** [comment]
> my values are wrong in the table, what can i do to fix it?

Fix the bug.

**Comment by LeoraBennett** [comment]
where is the bug?

**Comment by andrewoid** [comment]
> where is the bug?

Did you work with anyone else on this code?

**Comment by LeoraBennett** [comment]
I did what I could by myself and got a little help from peers on fixing the for loop. Is that a problem? On Feb 28, 2024, at 4:23 PM, Andrew Schwimmer ***@***.***> wrote:﻿

where is the bug?

Did you work with anyone else on this code?

**Comment by andrewoid** [comment]
> I did what I could by myself and got a little help from peers on fixing the for loop. Is that a problem? 

Yes.
Another student had the exact same bug as you. It's actually a difficult bug to write and I find the chances of two of you to randomly have the exact same bug to be very small.

**Comment by LeoraBennett** [comment]
Oh ok so maybe it is from that part I got help with. Next time I will reach out directly for help if that is better. What’s best for me to do now?Thank youOn Feb 28, 2024, at 4:35 PM, Andrew Schwimmer ***@***.***> wrote:﻿

I did what I could by myself and got a little help from peers on fixing the for loop. Is that a problem? 

Yes.
Another student had the exact same bug as you. It's actually a difficult bug to write and I find the chances of two of you to randomly have the exact same bug to be very small.

**Comment by andrewoid** [comment]
> Oh ok so maybe it is from that part I got help with. Next time I will reach out directly for help if that is better. What’s best for me to do now?

Find the bug.

File: src/main/java/bennett/projectile/ProjectileGraph.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                    (int) projectile.getX(),
+                    -(int) projectile.getY());
+        }
+
+        g.setColor(Color.BLUE);
+        g.fillOval(
+                ((int) projectile.getX() / 2) - 6,
+                (int) -projectile.getPeakY() - 4,
+                10,
+                10
```
Comment (andrewoid) [inline src/main/java/bennett/projectile/ProjectileGraph.java:55]: Oval dimentions
width = 10
height = 10

fillOval(x - 5, y - 5, 10, 10)

**Comment by andrewoid** [comment]
Please add a screenshot.

**Comment by LeoraBennett** [comment]
ProjectileGraph

**Pull request merged**